tsconfig.json

```diff
@@ -1,15 +1,14 @@
 {
     "compilerOptions": {
-      "target": "es5",
+      "target": "es2021",
+      "module": "ESNext",
       "allowJs": false,
       "strict": true,
       "noEmit": true,
       "esModuleInterop": true,
       "moduleResolution": "node",
-      "resolveJsonModule": true,
-      "isolatedModules": true,
       "incremental": true,
-      "downlevelIteration": true
+      "skipLibCheck": true
```

Comment:
I couldn't get this error working:

```
node_modules/micromark-extension-mdxjs/index.d.ts:7:14 - error TS2307: Cannot find module './node_modules/micromark-extension-mdx-expression/dev/lib/syntax.js' or its corresponding type declarations.

7     | import('./node_modules/micromark-extension-mdx-expression/dev/lib/syntax.js').Options
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/micromark-extension-mdxjs/index.d.ts:7
```

Using a declaration file didn't work. Tomorrow, I'm going to try to get a minimal reproduction and file an issue upstream.